### ISSUE TITLE
Official Recommendations

### ISSUE THREAD

**Issue by viluon** [oeed/CraftOS-Standards#6]
Maintain a list of preferred (recommended) formats for different categories. Multiple formats for the same category can be recommended. For a standard to be recommended, it has to be accepted and proven to be used by a large enough number of users (this might be very hard to determine, but I'll leave this up on the contributors and owners of this repository :stuck_out_tongue:). A good example of a standard to be recommended is [NFP](https://github.com/oeed/CraftOS-Standards/blob/master/File%20Formats/Images/Paintutils%20Image%20Format.md), the default image format for `paint` files.

**Comment by oeed** [comment]
Yeah that is a good point. Maybe mentioning it in the category README. I'd say we should only be accepting multiple standards for one thing (i.e. image formats) if they have different use cases or they're all highly used.

**Comment by viluon** [comment]
_Bump_, recommendations need an explanation on what they are. A list of recommended standards can already be viewed [here](https://github.com/oeed/CraftOS-Standards/pulls?utf8=%E2%9C%93&q=label%3Arecommended) (although it was me who just labeled the recommended standards, which were listed in the [Image README](https://github.com/oeed/CraftOS-Standards/blob/master/File%20Formats/Images/README.md), and it _wasn't me_ who put them there, so please keep these lists **synced**, it's very important), but we should create a file about official recommendations maybe in the `/Tutorial/` folder (currently only in the `tutorial` branch). The rules for accepting a standard as `recommended` should also be outlined more clearly.
